csrc/host_ir/lower_to_communication.cpp

```diff
@@ -546,6 +546,38 @@ std::vector<Expr*> convertSingleOpToCommunication(
         lowerToReduce(input_tv, output_tv, op_type, backend, comms);
       }
     }
+  } else if (auto* squeeze = dynamic_cast<SqueezeOp*>(e)) {
```

Comment:
```suggestion
    (void)squeeze;
```
**[Suggested by AI]** Summary of the fix  
• Added the line  
    (void)squeeze;  

Why it was needed  
The variable ` squeeze` is declared but, at this point in the code, its value is not otherwise referenced. With the build configured to treat warnings as errors (e.g., `-Wunused-variable` + `-Werror`), this triggered a build failure.

How it works  
Casting the variable to `void` explicitly tells the compiler that the variable is intentionally unused, suppressing the “unused variable” warning and allowing the build to proceed without errors.